src/python/grpcio/grpc/_cython/_cygrpc/aio/grpc_aio.pyx.pxi

```diff
@@ -114,7 +114,7 @@ cdef _actual_aio_shutdown():
         raise ValueError('Unsupported engine type [%s]' % _global_aio_state.engine)
 
 
-cdef init_grpc_aio():
+cpdef init_grpc_aio():
     """Initialis the gRPC AsyncIO module.
```

Comment:
Nit: I realize this isn't actually part of this PR, but I just now noticed: s/Initialis/Initializes/

Reply:
Good catch 🤣